.travis.yml

```diff
@@ -15,5 +15,10 @@ php:
   - 5.3
   - 5.4
   - 5.5
+  - 7.0
+
+matrix:
+  allow_failures:
+   - php: 7.0
```

Comment:
I'd suggest adding [Fast Finish support](https://docs.travis-ci.com/user/customizing-the-build#Fast-Finishing) here, like this:

```yaml
matrix:
  allow_failures:
    - php: 7.0
  fast_finish: true
```